inc/saiacl.h

```diff
@@ -1997,7 +2073,8 @@ typedef struct _sai_acl_api_t
     sai_remove_acl_range_fn             remove_acl_range;
     sai_set_acl_range_attribute_fn      set_acl_range_attribute;
     sai_get_acl_range_attribute_fn      get_acl_range_attribute;
-
+    sai_create_acl_table_group_fn       create_acl_table_group;
+    sai_remove_acl_table_group_fn       remove_acl_table_group;
```

Comment:
@zubinshah @lguohan 

Consider an application which sends multiple rules dynamically and below is a combination we want to achieve. 

 MAC1  with priority P1 on port 1, port 2  
 MAC2 with priority P2 on port 2, port 3

Scenario 1 :

Create ACL table T1 – Qualifier Mac1, Priority 1 – Bind to ports P1 and P2

Create ACL table T2 – Qualifier Mac2, Priority 2 - Bind to ports P2 and P3

Scenario 2:

Create ACL table T1 – Qualifier Mac1, Priority 1 – Bind to port P1

Create ACL table T2 – Qualifier Mac2, Priority 2 - Bind to port P3

Create ACL table T3 with 2 entries – Qualifier Mac1, Priority 1 – Bind to port P2  ---- Here priority will become entry priority.
                                                                  Qualifier Mac2, Priority 2 

Scenario 3:

Create ACL table T3  with 2 entries  ---- Bind to switch bind point 
            Qualifier Mac1, Priority 1 –   Inports list will have ports P1 and P2                          
            Qualifier Mac2, Priority 2 – Inports list will have ports P2 and P3.

What is the recommendation from you for the above scenario? 
In the community meeting on Thursday Scenario #3 was recommended.   Please provide your thoughts on this.